### DISCUSSION TITLE
擷取螢幕

### DISCUSSION THREAD

**Discussion by NcuJerryKao** [1101-GS3073-C/Python-Community-GS3073-C#524] [Show and tell]
前幾天跟同學討論 對方竟然不會擷取螢幕 而這件事是我幾乎每天都會用到的
1. 搜尋 「剪取工具」：要自己拖曳 決定要傑的範圍 記得要按存檔喔
2. Windows 鍵 + Shift + S 一樣要自己拖曳存檔 基本上跟剪取工具一模一樣 僅看個人習慣
3. Windows 鍵 + G 可以看到有截圖根錄影此二功能 都會自動幫你存檔喔 只是要注意一下 存取的位置 我的話是 本機 > 影片 > 擷取

**Comment by brian431** [comment]
第二個應該是最好用的，不過要錄影的話還是要用第三個。然後你的同學應該要學習如何Google。

**Comment by Ben6uo** [comment]
第二個擷取螢幕很方便 但不知道擷完後放在那了

**Reply by NcuJerryKao** [reply]
截完後 要按檔案 並且另存新檔喔 
跟第一個一樣 需要手動存檔

**Comment by gnsJhenJie** [comment]
macOS 的話可以用 `command` + `shift` + `3`或 `command` + `shift` + `4` 來達成

**Reply by chia-kai-chang** [reply]
感謝，我很需要

**Comment by todoroki0910** [comment]
其實在看到這個討論前我也是不會的那個

**Comment by awkward-willy** [comment]
Win 11 後可以設定讓 Print Screen 選取特定區塊截圖，但 win+shift+s 的截圖方式還是偶有它的用處！

**Comment by williamyen0623** [comment]
我都按windows鍵，然後搜尋剪取工具好哈哈，快捷鍵都記不住

**Comment by BingRong1217** [comment]
很實用的方法！

**Comment by Aerofe** [comment]
錄影的話有Nvidia的GeForce Experience 那個蠻用的，他可以即時擷取前五分鐘的電腦畫面，很實用

**Comment by limitx0** [comment]
第三個方法可以直接按 `Win` + `Alt` + `G` 錄影
`Win` + `Alt` + `PrtScn` 截圖

**Reply by chia-kai-chang** [reply]
感謝同學補充，感覺可以寫到wiki裡面啊，雖然目前沒有計分，但是有空的同學幫整理一下吧

@NcuJerryKao 樓主可以幫整理一下嗎? 貼到wiki：https://github.com/NCU-GS3073-AI-Introduction/ai-introduction-community/wiki

 還有同樣的內容也幫忙貼到Python班的wiki吧，這是很實用的資訊啊：https://github.com/NCU-GS4719-Python/Python-Community/wiki

**Comment by Raywei0214** [comment]
我自己也算蠻常用window+G的，但還是要看電腦性能，像我i9設定60fps打APEX不會卡，剪取工具我還以為每個人都知道

**Comment by Raywei0214** [comment]
另外我還要補充一個同樣是擷取螢幕的快捷鍵，crtl+prtsc (鍵盤右上角) 然後在crtl V直接複製到留言欄之類的 ppt也可以，或者是window+prtsc 會直接存檔到 本機-螢幕擷取 的資料夾裡面 但這跟擷取工具就有點落差 這個是整個螢幕都截取 所以還需要裁剪之類的

**Comment by bww0401** [comment]
我以前也都是搜尋剪取工具截圖，直到某次被朋友笑我才改掉...

**Comment by awehao** [comment]
其實電腦還是有很多實用的快捷鍵的，隨著使用電腦的頻率越來越高，這些都會漸漸熟悉

**Comment by larry40305** [comment]
直接按PrtSc也能截

**Comment by Xuewen628** [comment]
螢幕截圖是我做報告的好幫手ㄟ
許多需要製作過程的報告都會用到它
還好我的電腦有內建快捷鍵 一件截圖
